index.js

```diff
@@ -115,23 +118,63 @@ class Turtler {
         // Create pad of empty spaces to match the width of this value to max width of this column
         return value + ' '.repeat(columnWidths[i] - value.length);
       // join on the pipe which denotes the border of a table element in markdown
-      }).join(' | ');
+      }).join(constants.columnSeparatorDefault);
 
       table += `| ${row} |\n`;
 
       if(l === 0) {
         // we add the header with the width of the column minus the addition of a pipe symbol
         table += columnWidths.map((width, i) => {
           if(i == 0) {
-            return '|' + '-'.repeat(width + 2) + '|';
+            return '|' + constants.markdownHeaderSeparotrDefault.repeat(width + 2) + '|';
           }
-          return '-'.repeat(width + 2) + '|';
+          return constants.markdownHeaderSeparotrDefault.repeat(width + 2) + '|';
         }).join('') + '\n';
       }
     });
 
     return table;
   }
+  /**
+   * will return an html table representation of the data
+   * @method html
+   * @return {String} - html table
+   */
+  html() {
+    if (!this.htmlTable) {
```

Comment:
same comment as the other PR. This could be a lot cleaner with the positive case short stopping any code execution first, removes the extra indents needed.